src/browser-realm.ts

```diff
@@ -252,6 +252,7 @@ export default function createSecureEnvironment(distortionMap?: Map<RedProxyTarg
                 // in case the constructor inference fails
                 blueError = ErrorCreate(message);
             }
+            env.setRefMapEntries(e, blueError);
```

Comment:
in the first line inside the try, you need to use `env.getBlueValue` instead of ref.

Reply:
If we use `env.getBlueValue` red error objects will leak their properties into blue. The tests I wrote (now fixed) uncovered that behavior and it wasn't right. So I had to use env.getRedValue in red and env.getBlueRef in blue to avoid leaking things.